lib/traits/RenderableView.js

```diff
@@ -91,26 +91,46 @@ var RenderableView = _.extend({},
         },
 
         renderUnrenderedChildViews: function() {
-            _.each(this.unrenderedChildViews, function(childView) {
-                childView.renderChildViewIntoParent();
-            });
+            if (!this.unrenderedChildViews || !this.unrenderedChildViews.length) {
```

Comment:
the code in the for loop wont run if the length == 0 but the rest of the function body will. it's probably not a big deal that length is set unnecessarily. i'll check if there's any perceptible difference in performance